### ISSUE TITLE
Add issue template

### ISSUE THREAD

**Issue by gegoune** [nvim-tree/nvim-web-devicons#457]
To minimise number of round trips between reporters and maintainers we should introduce issue template to this repository.

I think very good example, which we should use in my opinion, is one from [Lewis' gitsigns plugin](https://github.com/lewis6991/gitsigns.nvim/tree/main/.github/ISSUE_TEMPLATE).

**Comment by alex-courtis** [comment]
Absolutely... I've been asking a lot lately...

That looks nice and small.

**Comment by gegoune** [comment]
Also avoids packer, which is not maintained any more.

**Comment by alex-courtis** [comment]
Oh damn... I liked packer...

Not going to go with lazy though ;)

**Comment by alex-courtis** [comment]
The approach of "git clone, no plugin manager" is probably best for nvim-tree too.

**Comment by gegoune** [comment]
Yes, it is. I would like to add the same template to nvim-tree too.

There is [pckr.nvim](https://github.com/lewis6991/pckr.nvim) that I happily use. Very similar experience to packer. pckr started as a refactor of packer.

**Comment by alex-courtis** [comment]
Oh nice, lewis... a respected member of the community.

I'll give it a go.
